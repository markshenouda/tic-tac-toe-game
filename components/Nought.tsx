import Svg, { Circle } from "react-native-svg";

export default function Nought() {
  return (
    <Svg width="100%" height="100%" viewBox="0 0 24 24" fill="none">
      <Circle
        cx="12"
        cy="12"
        r="9"
        stroke="white"
        strokeWidth="2.5"
        fill="none"
      />
    </Svg>
  );
}
