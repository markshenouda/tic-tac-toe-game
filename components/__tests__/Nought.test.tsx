import { render } from "@testing-library/react-native";
import * as React from "react";

import Nought from "@/components/Nought";

describe("Nought", () => {
  it("renders correctly", () => {
    const { toJSON } = render(<Nought />);
    expect(toJSON()).toMatchSnapshot();
  });
});
