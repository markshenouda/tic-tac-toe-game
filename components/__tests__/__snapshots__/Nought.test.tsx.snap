// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nought renders correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="100%"
  bbWidth="100%"
  fill="none"
  focusable={false}
  height="100%"
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": "100%",
        "width": "100%",
      },
    ]
  }
  vbHeight={24}
  vbWidth={24}
  width="100%"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGCircle
      cx="12"
      cy="12"
      fill={null}
      propList={
        [
          "fill",
          "stroke",
          "strokeWidth",
        ]
      }
      r="9"
      stroke={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      strokeWidth="2.5"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
